.github/ISSUE_TEMPLATE/component-request.md

```diff
@@ -7,59 +7,43 @@ assignees: ''
 
 ---
 
-<!--**Mandatory steps to ensure alignment between stakeholders and the progression of Kirby**-->
+<!-- 
+Explanation of applied labels can be found here: https://github.com/kirbydesign/designsystem/labels
+-->
 
-<!--In order to ensure steady progress and quality of Kirby, please follow our outlined process. By default four labels are added to new component issues and enhancements. To help Kirby please follow these steps, and remove the labels from the issue when done.-->
+## Describe the solution you'd like
+[ Replace this paragraph with a clear and concise description of what you want to happen ]
 
-<!--*New*-->
-<!--Indicates that this is a new issue that has not yet been addressed by the Kirby team. The `New` label will be removed by the Kirby team. -->
+## Have you considered any alternatives?
+[ Replace this paragraph with a clear and concise description of any alternative solutions or features you've considered (if any) ]
 
-<!--*NOT Prioritized*-->
-<!--Describe any deadlines for the issue - eg. X needs this done by Y date, to be used in Z sprint. Suggest a milestone for the issue. The `Not Prioritized` label will be removed by the Kirby team. -->
+## Are there any additional context?
+\[ Replace this paragraph with any additional context or screenshots about the component request here, e.g. a link to a corresponding component from [Ionic](https://ionicframework.com/docs/components/) (if any) ]
 
-<!--*NOT UX Refined*-->
-<!--Make sure the new Component, has a name, can be found in Zeplin, and is used in minimum one reviewed screen. Remove the `NOT UX Refined` label and add links to Zeplin.-->
+<hr />
 
-<!--*NOT Tech Refined*
-<!--Sketch a solution in technical terms, that is how will the component be build - eg. build it from scratch or build using X Ionic component. Call for a brief meeting or spend enough time with someone from @kirbydesign/kirby-guild to get a "go ahead". Remove the `NOT Tech Refined` label.-->
+## Checklist:
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Refinement
 
-**Additional context**
-Add any other context or screenshots about the feature request here, e.g. a link to the corresponding component from [Ionic](https://ionicframework.com/docs/components/)
+- [ ] Request that the issue is [UX refined](../CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
+- [ ] Request that the issue is [tech refined](../CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.
+- [ ] Move the issue to the "Ready to do"-column in the "[Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)"-project.
```

Comment:
```
- [ ] Move the issue to the "Ready to do"-column in the "[Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)"-project.
```
Do we really want contributors doing this - or should this step perhaps be exclusive for Team Kirby members. 
I.e. let's remove this from the checklists?

If we want to remove it, remember to also remove it from the issue templates.

Reply:
I agree that it is just something we should take care of.